chore(meta): Add details about alt text guidelines (#22603)
mdn/content: files/en-us/mdn/writing_guidelines/howto/images_media/index.md

## Adding alternative text to images

Every image, `![]` and `<img>`, must include `alt` text.
Alt attributes should be short, providing all the relevant information the image conveys.

> When writing an alt attribute, think about the valuable information of the image and how you would relay that information to someone who can read the page's content but can't load images.

Reviewer: ```suggestion
When writing the image description, think about the valuable information of the image and how you would relay that information to someone who can read the page's content but can't load images.
```

> Make sure to create an alt attribute for an image based on its context.

Reviewer: ```suggestion
Make sure the alternative text for the image is based on its context.
```

If the photo of Fluffy the dog is an avatar next to a review for Yuckymeat dog food, `alt="Fluffy"` is appropriate.
If the same photo is part of Fluffy's animal rescue adoption page, the information conveyed in the image is relevant for prospective dog parents, such as `alt="Fluffy, a tri-color terrier with very short hair, with a tennis ball in her mouth."`.

> The surrounding text likely has Fluffy's size and breed, and you would not need to include it as it would be redundant.

Reviewer: ```suggestion
The surrounding text likely has Fluffy's size and breed, so including it would be redundant.
```
